core/src/main/scala/com/rallyhealth/vapors/factfilter/dsl/LogicalSyntax.scala

```diff
@@ -0,0 +1,29 @@
+package com.rallyhealth.vapors.factfilter.dsl
+
+import com.rallyhealth.vapors.factfilter.dsl.ExprDsl.CondExpr
+
+trait LogicalSyntax {
+
+  // TODO: Use ExtractBoolean?
+  implicit def booleanOps[F[_], V, P](expr: CondExpr[F, V, P]): BooleanSyntaxOps[F, V, P] = new BooleanSyntaxOps(expr)
+}
+
+class BooleanSyntaxOps[F[_], V, P](private val expr: CondExpr[F, V, P]) extends AnyVal {
+
+  def unary_!(
+    implicit
+    post: CaptureP[F, V, Boolean, P],
+  ): CondExpr[F, V, P] = ExprDsl.not(expr)
+
+  def ||(
+    other: CondExpr[F, V, P],
+  )(implicit
+    post: CaptureP[F, V, Boolean, P],
+  ): CondExpr[F, V, P] = ExprDsl.or(expr, other)
+
+  def &&(
+    other: CondExpr[F, V, P],
+  )(implicit
+    post: CaptureP[F, V, Boolean, P],
+  ): CondExpr[F, V, P] = ExprDsl.and(expr, other)
+}
```

Comment:
I'm not sure this syntax is even used with the new builder pattern. I should probably move it to the ExprBuilder or at least make it cross compatible.

Reply:
Removed.